### DISCUSSION TITLE
PaddleOCR/latest/ppocr/infer_deploy/paddle_server

### DISCUSSION THREAD

**Discussion by giscus** [PaddlePaddle/PaddleOCR#13986] [Q&A]
# PaddleOCR/latest/ppocr/infer_deploy/paddle_server

Awesome multilingual OCR toolkits based on PaddlePaddle (practical ultra lightweight OCR system, support 80+ languages recognition, provide data annotation and synthesis tools, support training and deployment among server, mobile, embedded and IoT devices)

https://paddlepaddle.github.io/PaddleOCR/latest/ppocr/infer_deploy/paddle_server.html

**Comment by Licyldj** [comment]
serving c++ 支持 OCRv4 模型吗。

**Reply by Liyulingyue** [reply]
建议参考 https://github.com/PaddlePaddle/PaddleX/blob/release/3.0-beta1/docs/pipeline_deploy/service_deploy.md
